redbot/core/dev_commands.py

```diff
@@ -242,17 +243,20 @@ async def repl(self, ctx):
             )
```

Comment:
Maybe you could mention that you can "unpause" the REPL using `[p]repl pause`, if it's paused, so that it's less confusing to the user why they can use it even though it's running.